Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.7.3-slim-stretch
+RUN apt-get -y update && apt-get -y install gcc
```

```suggestion
ENV DEBIAN_FRONTEND noninteractive
RUN apt-get -qq update && \
    apt-get -qq install gcc g++ && \
    apt-get --purge autoremove --yes && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```
it helps to use fewer docker layers to reduce image size, also to remove unused files

---

Makes sense, thanks.